schemas/user.py

```
@@ -0,0 +1,59 @@
+    profile_img: str | None = None
+    created_at: datetime
+
+
+class UserUpdateRequest(BaseModel):
+    nickname: str | None = None
+    profile_img: str | None = None
+
+    @model_validator(mode='after')
+    def check_at_least_one_field(self):
+        if self.nickname is None and self.profile_img is None:
+            raise AttributeError("최소 하나의 필드는 입력해야 합니다")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)